### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by Rajrahane**
Added sample Data and output
Highlighted certain points

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,37 +1,40 @@
...
+# Swasthya-AI-Placement-GraphQL-Server
 GraphQL server for Swasthya AI Placement Assignment
 
 Visit the <a href='https://github.com/Rajrahane/Swasthya-AI-Placement-React-Server'>React Client Repository</a>
 
-Run the MYSQL script on MySQL Workbench<br>
+Run the MYSQL script dump_2329_22052021.sql on MySQL Workbench<br>
 Steps to install and run GraphQL and React Servers:<br>
 1. clone the repository (git clone repo.git)
 2. run npm install
```
Comment (visrahane) [inline README.md:9]: `npm install` - use backtick ` for commands

File: README.md [hunk 2]
```diff
@@ -1,37 +1,40 @@
...
-5. view the GraphQL server on localhost:4000/graphql
-6. view the React Server on localhost:3000/
+5. view the GraphQL server on http://localhost:4000/graphql
+6. view the React Server on http://localhost:3000/
 
-Approach:<br>
-Create 3 tables: User, Blog and Comment<br>
+<h4>Approach:</h4>
+<b>DB Schema:</b>
+Create 4 tables: User, Blog, Comment, Friend<br>
```
Comment (visrahane) [inline README.md:18]: use tables instead

**Review by visrahane — COMMENTED** [review]
Commented

File: README.md (hunk 2 above)
Comment (Rajrahane) [reply README.md:18]: resolved

**Pull request merged**